source/_docs/getting-support.md

```diff
@@ -19,21 +20,21 @@ Submit a ticket with our Support Team to request assistance on issues within our
  - The error message, if applicable.
  - Links to screenshots or screencasts of the behavior, if necessary.
 
-##Other Ways to Contact Support
+## Other Ways to Contact Support
 If you cannot sign in to your Pantheon Dashboard:  
 
 - Use the [Password Reset](https://dashboard.pantheon.io/reset-password) form for Pantheon Dashboard access
 - Visit our [Contact form](https://pantheon.io/contact-us) and choose **I can't login and I need support**
 - Email Support directly at helpdesk@pantheon.io or cse-support@pantheon.io
 
+## Customer Support Response Times
 
-## Service Level Agreement (SLA) for Downtime
-Support tickets filed by Elite plans have a first acknowledgement of receipt guarantee:  
-
-* Downtime ticket: 30 minutes  
-* Standard support ticket: 2 hours
+###Elite Plans and Priority Enterprise Support Customers    
+* **On-call ticket**: Available 24/7/365 with a guaranteed response time guarantee for business critical issues that include downtime, development team blockers, and urgent deadline-driven questions. For issues requiring an on-call engineer, we guarantee escalation in 60 minutes.
+* **Standard support ticket**: Your general support questions are routed through our support queue ahead of other users. Expedited support is accessed through our Dashboard and email.

```

Comment:
Just quibble on language: here the phrase is "standard support ticket" and below it is "normal ticket". I think it would be clearer if we had consistent language. 

Another quibble: "guaranteed response time guarantee" is redundant.